oauthlib/oauth2/rfc6749/grant_types/base.py

```diff
@@ -17,6 +18,75 @@ class GrantTypeBase(object):
     error_uri = None
     request_validator = None
     default_response_mode = 'fragment'
+    refresh_token = True
+    response_types = ['code']
+
+    def __init__(self, request_validator=None, **kwargs):
+        self.request_validator = request_validator or RequestValidator()
```

Comment:
Well, you could pass `_auth_validators_run_before_standard_ones` as a keyword argument to the constructor, and that would do it. But that feels a bit hacky.